package.json

```diff
@@ -24,7 +25,7 @@
     "build": "npm run build:exports && npm run build-react-icons && webpack --mode production --config webpack/default.config.js",
     "build:exports": "./scripts/build-exports.js",
     "build:ts-index": "./scripts/build-ts-index.js",
-    "storybook": "start-storybook -p 7007",
+    "storybook": "start-storybook -p 7008",
```

Comment:
why?

Reply:
Just so it be more convenient - monday-ui-components also runs on localhost:7007 and when u r running them together u need to make more actions like 'press y, so it would run on a different port'...
Decided that it's better to change it here